web/modules/custom/do_base/do_base.deploy.php

```
@@ -8,3 +8,225 @@
+    if ($html === FALSE) {
+      continue;
+    }
+
+    $paragraph = Paragraph::create([
+      'type' => 'civictheme_content',
+      'field_c_p_theme' => 'dark',
+      'field_c_p_content' => [
+        'value' => trim($html),
+        'format' => 'full_html',
+      ],
+    ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\paragraphs\Entity\Paragraph' in method '_do_base_html_paragraphs'. (undefined)

(StaticAccess)